src/test/postman/Document_Signing_API.postman_collection.json

```diff
@@ -0,0 +1,294 @@
+{
+	"info": {
+		"_postman_id": "d922a2a7-a2e0-497a-a702-9f8cdd6c5b50",
+		"name": "Document Signing API",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "7646296"
+	},
+	"item": [
+		{
+			"name": "sign PDF",
+			"item": [
+				{
+					"name": "sign PDF fairweather",
+					"request": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n    \"document_location\": \"https://document-api-images-cidev.s3.eu-west-2.amazonaws.com/9616659670.pdf\",\n    \"document_type\": \"certified-copy\",\n    \"signature_options\": [\n        \"cover-sheet\"\n    ]\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{base_url}}/document-signing-api/document-signing/sign-pdf",
+							"host": [
```

Comment:
formatting in this file seems to use tabs instead of spaces

Reply:
I'll look into this.